modules/_labs/drawer/react/spec/Drawer.spec.tsx

```diff
@@ -0,0 +1,29 @@
+import * as React from 'react';
+import {render, fireEvent} from '@testing-library/react';
+
+import {Drawer, DrawerHeader, DrawerDirection} from '../index';
+
+describe('Drawer', () => {
+  test('should call a callback function', async () => {
+    const cb = jest.fn();
+    const {findByLabelText} = render(
+      <Drawer
+        header={
+          <DrawerHeader iconLabel={'Close'} title={'Header Title'} onClose={cb}></DrawerHeader>
+        }
+      >
+        Hello World
+      </Drawer>
+    );
+
+    fireEvent.click(await findByLabelText('Close'));
+
+    expect(cb).toHaveBeenCalledTimes(1);
+  });
+
+  test('Modal should spread extra props', async () => {
```

Comment:
Remove the word "Modal" here (it's already in a "Drawer" describe block)

Reply:
bad copy pasta